src/qilib/data_set/data_array.py

```diff
@@ -35,7 +35,7 @@ class DataArray:
     """
 
     def __init__(self, name: str, label: str, unit: str = '', is_setpoint: bool = False,
-                 preset_data: Optional[numpy.ndarray] = None, set_arrays: Optional[List['DataArray']] = None,
+                 preset_data: Optional[numpy.ndarray] = None, set_arrays: Optional[Union[List['DataArray'], Tuple['DataArray', ...], 'DataArray']] = None,
```

Comment:
After the type hinting, this line looks pretty lengthy. Can we do something about it?

Reply:
fix it, and, it's fixed